Extension/src/LanguageServer/ui.ts

```diff
@@ -171,7 +171,7 @@ export class UI {
         for (let i: number = 0; i < configurationNames.length; i++) {
             items.push({ label: configurationNames[i], description: "", index: i });
         }
-        items.push({ label: "Edit Configurations...", description: "", index: configurationNames.length });
+        items.push({ label: "Edit Configurations (JSON)", description: "", index: configurationNames.length });
```

Comment:
This changes it for every invocation, though, right?  We don't want to force users to go to JSON when they click on the label in the status bar, or run the command to select a config manually.

Reply:
I think we want to register a new private command (not in package.json).  An example:
https://github.com/Microsoft/vscode-cpptools/blob/f3a02fde4e466991230a6a55cc92456cb07bc5dd/Extension/src/LanguageServer/extension.ts#L680

The command will behave almost identically to the `select` function, but will open the JSON file instead of the UI regardless of the user's UI vs JSON setting.